lib/TeamcityReporter.js

```diff
@@ -34,10 +34,10 @@ class TeamcityReporter {
     item(err, args) {
         if (!this.currItem.passed) {
             const msg = this.escape(this.currItem.failedAssertions.join(", "));
-            const responseCode = (this.currItem.response && this.currItem.response.responseTime) || "-";
+            const responseCode = (this.currItem.response && this.currItem.response.code) || "-";
             const reason = (this.currItem.response && this.currItem.response.reason()) || "-";
             const details = this.escape(`Response code: ${responseCode}, reason: ${reason}`);
-            console.log(`##teamcity[testFailed name='${this.currItem.name}' message='${msg}' details='${msg} - ${details}' flowId='${this.flowId}']`);
+            console.log(`##teamcity[testFailed name='${this.currItem.name}' details='${msg} - ${details}' flowId='${this.flowId}']`);
```

Comment:
Teamcity uses the `message` in some cases without the `details`, for instance, in a list with a succinct description. For that reason we shouldn't remove the message attribute. You might have a point about removing `${msg}` from `details` unless there is view in the TC UI where `details` are displayed without `message`.

This change is independent of the fix you are making above though, so let's put it in a different commit so I can accept the fix.

Reply:
That is fair enough, thanks for your comments. My purpose is just to remove the duplicated {msg} log info, so removing the {msg} from "details='${msg} - ${details}'" could be a better solution considering your concern. For now, I just change the responseCode in the new commit.